app.js

```diff
@@ -45,5 +66,16 @@ var qual = document.getElementsByName("c");
       }).catch(function(error) {
         console.log(error);
       });
+    }
+
+      window.location.href="success.html";

```

Comment:
proper routing has to be used for the page navigation

Reply:
issue created: #8